must-gather/node-gather/daemonset.yaml

```diff
@@ -64,3 +64,4 @@ spec:
         hostPath:
           path: /var/lib/kubelet/pod-resources
           type: Directory
```

Comment:
if you re-upload, please remove this change - it's unnecessary

Reply:
Done, sorry, its was a copy paste issue